tidb-cloud/changefeed-sink-to-cloud-storage.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Sink to Cloud Storage
+Summary: Learn how to create a changefeed to stream data from a TiDB Dedicated cluster to cloud storage, such as Amazon S3 etc.
+---
+
+# Sink to Cloud Storage
+
+This document describes how to create a changefeed to stream data from TiDB Cloud to cloud storage. Currently, only Amazon S3 is supported.
+
+> **Note:**
+>
+> To stream data to cloud storage, make sure that your TiDB cluster version is v7.1.0 or later.
+>
+> For [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless-beta) clusters, the changefeed feature is unavailable.
+
+## Restrictions
+
+- For each TiDB Cloud cluster, you can create up to 5 changefeeds.
+- Because TiDB Cloud uses TiCDC to establish changefeeds, it has the same [restrictions as TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview#unsupported-scenarios).
+- If the table to be replicated does not have a primary key or a non-null unique index, the absence of a unique constraint during replication could result in duplicated data being inserted downstream in some retry scenarios.
+
+## Create a changefeed
+
+1. Navigate to the cluster overview page of the target TiDB cluster, and then click **Changefeed** in the left navigation pane.
+
+2. Click **Create Changefeed**, and select **Amazon S3** as the destination.
+
+3. Fill in the fields in the **S3 Endpoint** area: `S3 URI`, `Access Key ID` and  `Secret Access Key`.
+
+    ![create changefeed to sink to s3](/media/tidb-cloud/changefeed/sink-to-s3-01-create-changefeed.jpg)
+
+4. Click **Next** to establish the connection from the TiDB Dedicated cluster to Amazaon S3. TiDB Cloud will automatically test and verify if the connection is successful.
+
+    - If yes, you are directed to the next step of configuration.
+    - If not, a connectivity error is displayed, and you need to handle the error. After the error is resolved, click **Next** again.
+
+5. Customize **Table Filter** to filter the tables that you want to replicate. For the rule syntax, refer to [table filter rules](https://docs.pingcap.com/tidb/stable/ticdc-filter#changefeed-log-filters).
+
+    ![the table filter of changefeed](/media/tidb-cloud/changefeed/sink-to-s3-02-table-filter.jpg)
+
```

Comment:
```suggestion

```

Reply:
Since the console UI of TiDB Cloud changes frequently, continually updating the screenshots in our documentation becomes time-consuming. The "Create Changefeed" UI design is self-explanatory and it's easy for users to follow even without screenshots. So, we recommend removing screenshots in this document to simplify maintenance.